src/StackExchange.Redis/StackExchange.Redis.csproj

```diff
@@ -3,7 +3,8 @@
     <Nullable>enable</Nullable>
     <!-- extend the default lib targets for the main lib; mostly because of "vectors" -->
     <!-- Note: we're NOT building for .NET 6 because the thread pool changes are not wins yet -->
-    <TargetFrameworks>net461;netstandard2.0;net472;netcoreapp3.1;net5.0</TargetFrameworks>
+    <!-- <TargetFrameworks>net461;netstandard2.0;net472;netcoreapp3.1;net5.0</TargetFrameworks> -->
+    <TargetFrameworks>netstandard2.0;net472;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
was too lazy to install net461

Reply:
FWIW, you don't have to install it to build for it - packages pull it in via NuGet :)